torch/csrc/jit/ir/scope.h

```diff
@@ -50,6 +51,26 @@ struct TORCH_API Scope : public c10::intrusive_ptr_target {
 
 struct Function;
 struct InlinedCallStack;
+struct ModuleInstanceInfo {
+ private:
+  c10::ClassTypePtr module_type_{nullptr};
```

Comment:
I'm not sure if you need to initialize it to `{nullptr}` if you are going to be assigning this in the member constructor list of the constructor.

Reply:
After a second thought, `c10::ClassTypePtr` doesn't have a default constructor and it might be error-prone when it's not used correctly. Having a default nullptr here is probably a safer option.